### PR TITLE
Hinchley2018/homepage

### PR DESCRIPTION
**Pull request by hinchley2018**
All the fixings, products list, styling, scripts, etc

### CODE REVIEW

**Comment by hinchley2018** [comment]
Closes #1

**Comment by hinchley2018** [comment]
@jvannier created the e-commerce site, LMK what you and PM think

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                <h3>Duct tape</h3>
+                <p><b>$4.72</b></p>
+                <img alt="duct tape" src="assets/duct-tape.png" />
+                <button  class="btn btn-secondary" onclick="addToCart('Duct tape',4.72, 'assets/duct-tape.png')">Add to Cart</button>
+            </li>
+            <li>
+                <h3>Macbook Pro</h3>
+                <p><b>$2,599</b></p>
+                <img alt="Macbook pro" src="assets/macbookpro.png"/>
+                <button   class="btn btn-secondary" onclick="addToCart('Macbook Pro', 2,599,'assets/macbookpro.png')">Add to Cart</button>
```
Comment (jvannier) [inline index.html:47]: nit: spacing

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            </li>
+            <li>
+                <h3>Mouse</h3>
+                <p><b>$29.99</b></p>
+                <img alt="Gaming Mouse" src="assets/mouse.png"/>
+                <button  class="btn btn-secondary"  onclick="addToCart('Mouse',29.99,'assets/mouse.png')">Add to Cart</button>
```
Comment (jvannier) [inline index.html:53]: nit: spacing. Class name could also be more descriptive

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+.navbar{
+    height:100px;
+    
+}
+
+
+.name {
+  
+    color:white;
+    font-size:40px;
```
Comment (jvannier) [inline styles.css:32]: Do we want this to be sized in relation to the page instead?

**Review by jvannier — APPROVED** [review]

**Pull request merged**